src/libcases.py

```diff
@@ -29,18 +35,15 @@ def run_libcase(item_dict, plot_option="all-compact"):
         need_injection = False
     run_sim = item.item["run_simulation"]
 
-    if need_injection and run_sim:
+    if need_injection:
```

Comment:
Writing unit tests (`run_sim == False`) it showed that the following section is also needed when dealing with EnergyPlus outputs but not running the simulation.